sculpt.js

```diff
@@ -7,7 +7,7 @@ const path = require('path')
 const Parse = require('fast-json-parse')
 
 const defaultOptions = {
-  path: os.tmpdir() + 'aborsculpt.json',
+  path: `${os.tmpdir()}${path.sep}aborsculpt.json`,
```

Comment:
please use `path.join()`

Reply:
Agreed. It's my fault for not doing it in the first place, but let's correct that mistake.

Reply:
No problem, but how to specify the test. As I wrote I can not imagine for a good test, because there is no way to inject something or am I wrong. The only possibility I see is to create a json, modify it and react to the modification?

Reply:
You can use http://npm.im/proxyquire to alter the required modules.

Reply:
Thank's for the advice.